sdk/eventhub/Azure.Messaging.EventHubs.Experimental/tests/Producer/IdempotentProducerLiveTests.cs

```diff
@@ -0,0 +1,364 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.WebSockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Amqp;
+using Azure.Messaging.EventHubs.Authorization;
+using Azure.Messaging.EventHubs.Consumer;
+using Azure.Messaging.EventHubs.Core;
+using Azure.Messaging.EventHubs.Producer;
+using Azure.Messaging.EventHubs.Tests;
+using NUnit.Framework;
+
+namespace Azure.Messaging.EventHubs.Experimental.Tests
+{
+    /// <summary>
+    ///   The suite of live tests for the <see cref="IdempotentProducer" />
+    ///   class.
+    /// </summary>
+    ///
+    /// <remarks>
+    ///   These tests have a dependency on live Azure services and may
+    ///   incur costs for the associated Azure subscription.
+    /// </remarks>
+    ///
+    [TestFixture]
+    [Category(TestCategory.Live)]
+    [Category(TestCategory.DisallowVisualStudioLiveUnitTesting)]
+    public class IdempotentProducerLiveTests
```

Comment:
There's a much more comprehensive set of tests in the core library.   These are aimed only at validating core scenarios to ensure the pass-though configuration is working as intended.